.github/workflows/update.yml

```diff
@@ -1,11 +1,30 @@
 name: update-flake-lock
 on:
   workflow_dispatch: # allows manual triggering
+    inputs:
+      branch:
+        description: "Branch to update"
+        type: choice
+        options:
+          - 'stable & unstable'
+          - 'main'
+          - 'nixos-23.11'
   schedule:
     - cron: "0 12 * * SAT" # runs weekly on Saturday at noon
 
 jobs:
   lockfile:
+    strategy:
+      matrix:
+        # This allows to update both stable & unstable branches, but not both when triggered
```

Comment:
Excluding this way seems a bit backwards, and won't scale if we ever want to add a third branch.

But it's probably not worth thinking up a better solution until we actually decide to do that.

Reply:
Yeah I agree, but that seems a bit of a limitation of github actions

Reply:
Can't look properly right now, but I think it'd be possible to use `${{ }}` expressions and `&&` `||` operators together to form a ternary.